packages/atomic/src/components.d.ts

```diff
@@ -58,7 +58,7 @@ export namespace Components {
          */
```

Comment:
Ignore this file, it looks like somebody merged doc changes without the `components.d.ts` file.